src/sage/matrix/matrix2.pyx

```diff
@@ -99,7 +99,7 @@ from sage.arith.numerical_approx cimport digits_to_bits
 from copy import copy
 
 import sage.modules.free_module
-from . import berlekamp_massey
```

Comment:
here we used `from . import berlekamp_massey` to import a module and it is replaced by `from sage.matrix import berlekamp_massey`. But in other places, for instance in `src/sage/matroids/unpickling.pyx`, we use `import sage.matroids.matroid` (not modified). I'm not sure of the difference between both forms. Which one is the most suited ?

Reply:
After `from sage.matrix import berlekamp_massey`, the module can be referred to as `berlekamp_massey`. This style tends to appear in older portions of the Sage source code. Personally I don't like it very much, but I have not made an effort to change it.

Reply:
I see.